### PR TITLE
modify text on taking liklihood separately for each sentence

### PR DESCRIPTION
**Pull request by udibr**
https://github.com/piskvorky/gensim/issues/514

### CODE REVIEW

File: docs/notebooks/deepir.ipynb [hunk 1]
```diff
@@ -333,7 +333,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Fo any new sentence we can obtain its _likelihood_ (lhd; actually, the composite likelihood approximation; see the paper) using the [score](https://radimrehurek.com/gensim/models/word2vec.html#gensim.models.word2vec.Word2Vec.score) function in the `word2vec` class.  We get the likelihood for each sentence in the first test review, then convert to a probability over star ratings.  This is all in the following handy wrapper."
+    "For any new sentence we can obtain its _likelihood_ (lhd; actually, the composite likelihood approximation; see the paper) using the [score](https://radimrehurek.com/gensim/models/word2vec.html#gensim.models.word2vec.Word2Vec.score) function in the `word2vec` class.  We get the likelihood for each sentence in the first test review, then convert to a probability over star ratings. Every sentence in the review is evaluated separately and the final star raring of the review is an average vote of all the sentences. This is all in the following handy wrapper."
```
Comment (piskvorky) [inline docs/notebooks/deepir.ipynb]: "raring" => "rating"?

**Comment by mataddy** [comment]
@udibr the new text looks great to me; thanks for clarifying! and for fixing my typos.
 
not sure why the build failed, but it has nothing to do with the notebook so this should be fine to merge.  cc @piskvorky

**Comment by piskvorky** [comment]
Yes, unrelated. That error is coming from new keywords test -- @tmylk please look into it.

**Comment by piskvorky** [comment]
Merging -- thanks a lot guys! Great to have such capable contributors :)

**Pull request merged**